translations/ui-users/en.json

```diff
@@ -367,7 +367,7 @@
   "readingRoom.name": "Name",
   "readingRoom.note": "Note",
   "readingRoom.lastUpdated": "Last updated",
-  "readingRoom.filter": "Filter",
+  "readingRoom.filter": "Room name",
   "readingRoom.allowed": "Allowed",
   "readingRoom.notAllowed": "Not allowed",
   "readingRoom.metaSection.lastUpdatedDetails": "{date} by {name}",
```

Comment:
Unrelated to this PR, but `{date} by {name}` should be sth like `{date, date, full} by {name}` or `{date, date, short} by {name}` etc. [dateStyle options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle).